Documentation/docs-site/content/usage/access-secret-cfgmap-in-function.md

```diff
@@ -1,8 +1,11 @@
 ---
-title: "Access secret/configmap in function"
+title: "Secret/configmap in function"
```

Comment:
I liked the previous title better (though "Accessing secrets..." is better than "Access secret...")

Reply:
Yeah, I think I shortened it because it was not looking good, most of the titles are short and this is much longer than others. For now, I changed it back, but I want to get back to aesthetics of this later :)
![image](https://user-images.githubusercontent.com/1167948/36521907-7ff2b1ac-17bf-11e8-91f6-ef4cd79283e0.png)